CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java

```diff
@@ -43,6 +43,7 @@
   String PCODE = null;
   String DOMAIN = null;
   private final String ACCOUNT_ID = "accountID";
+  Boolean autoPlay = false;
```

Comment:
Seems like you have some other or outdated version.
There is already an autoplay field in latest candidate.
![image](https://user-images.githubusercontent.com/43542852/49361952-5ac04700-f6ee-11e8-8030-0df2b10cfaf2.png)

Reply:
Ah, that's another Activity, ok